.github/workflows/lint.yml

```diff
@@ -41,12 +41,8 @@ jobs:
       with:
```

Comment:
This PR also removes the algol60 dependency in the workflows since this package is supposed to be independent of HPE resources.